lua/telekasten.lua

```diff
@@ -148,6 +148,17 @@ local function defaultConfig(home)
         -- "catimg-previewer" if you have catimg installed
         -- "viu-previewer" if you have viu installed
         media_previewer = "telescope-media-files",
+        -- files which will be aviable in insert and preview images list
+        media_extensions = {
+            ".png",
+            ".jpg",
+            ".bmp",
+            ".gif",
+            ".pdf",
+            ".mp4",
+            ".webm",
+            ".webp",
```

Comment:
You put `.webp` in the default configuration, but not in the documentation.
Likewise, you put `.svg` in the documentation but not in the configuration.

Could you sort this out so the documentation matches exactly the default config?

Reply:
Yes, I updated it